### DISCUSSION TITLE
Cant render new folder in content

### DISCUSSION THREAD

**Discussion by jennyberry** [thuliteio/doks#861] [Q&A]
I try to add new item in menu, so I create new folder in content. However, when I click on it, the website said "404 page not found" even there is a file in that folder.  
This is my file structure:
<img width="265" alt="Screen Shot 2022-08-03 at 11 33 55 pm" src="https://user-images.githubusercontent.com/64534681/182621038-a36dbd34-1f1e-482f-b422-f631d8be08d5.png">

What I did in `menu.toml`
<img width="615" alt="Screen Shot 2022-08-03 at 11 27 52 pm" src="https://user-images.githubusercontent.com/64534681/182620443-e40afd7b-1d16-432d-9b74-b0ff27b075e2.png">

 Please help, thanks in advance!

**Comment by h-enk** [comment]
Do you have a repo you could share?

**Reply by jennyberry** [reply]
It actually working now, I have no idea why this happen but thank you!
